core/trino-main/src/main/java/io/trino/operator/OperatorContext.java

```diff
@@ -182,7 +184,7 @@ public void recordPhysicalInputWithTiming(long sizeInBytes, long positions, long
     {
         physicalInputDataSize.update(sizeInBytes);
         physicalInputPositions.update(positions);
-        addInputTiming.record(readNanos, 0);
+        physicalInputReadTimeNanos.getAndAdd(readNanos);
```

Comment:
This changes semantics of `addInputTiming` that used to be updated here.
Why not update both?
Please explan.

Reply:
This was a bug. `addInputTiming` is still being correctly updated in `io.trino.operator.Driver#processInternal (                        next.getOperatorContext().recordAddInput(operationTimer, page);`) and by `WorkProcessorPipelineSourceOperator`

Reply:
Should bugfix be a separate commit then?

Reply:
This commit is a bugfix.